frontend/src/types/AuthTypes.ts

```diff
@@ -1,10 +1,11 @@
-export type AuthenticatedUser = {
```

Comment:
Is there a particular reason to rename in this way? It makes the diff pretty large and it would be nice to keep the unshortened version unless there's a concrete benefit to doing so.

Reply:
I understand that, it's just that it's a misnomer to say that this user will be necessarily authenticated because they may not be verified. Therefore, I think AuthUser is a better name for this. I can understand if you don't think it's worth it though.

Reply:
Let's keep it the same if possible, if you don't mind reverting everything back? If you feel strongly about it we can make a separate ticket and PR for that change.

Reply:
I'd also argue "authenticated" doesn't mean the same as "verified"? 

(If they've logged in successfully, they're still authenticated. Unverified email is a separate concept in my mind)

Reply:
> Let's keep it the same if possible, if you don't mind reverting everything back? If you feel strongly about it we can make a separate ticket and PR for that change.

Good to just have this happen in a separate PR.

Reply:
Would you be able to revert these name changes then, @mslwang ?

Reply:
sounds good